### PR TITLE
Troubleshooting and fixing manifest handling

### PR DESCRIPTION
**Pull request by kedhammar**

### CODE REVIEW

File: taca/element/Element_Runs.py [hunk 1]
```diff
@@ -379,44 +374,38 @@ def find_manifest_zip(self):
             logger.warning(
                 f"No manifest found for run '{self.run_dir}' with pattern '{glob_pattern}'."
             )
-            return False  # TODO: determine whether to raise an error here instead
+            return None
         elif len(glob_results) > 1:
             logger.warning(
                 f"Multiple manifests found for run '{self.run_dir}' with pattern '{glob_pattern}', using latest one."
             )
             glob_results.sort()
```
Comment (ssjunnebo) [inline taca/element/Element_Runs.py:382]: I guess this works as long as no-one renames the manifest files, which we do for illumina during redemux. Fine for now but we need to think about it later.

File: taca/element/Element_Runs.py [hunk 2]
```diff
@@ -493,15 +482,21 @@ def make_demux_manifests(
...
 
         # Break down into groups by non-consolable properties
-        grouped_df = df_samples.groupby(
-            ["I1Mask", "I2Mask", "UmiMask", "Lane", "Recipe"]
-        )
+        grouped_df = df_samples.groupby(["I1Mask", "I2Mask", "UmiMask", "Recipe"])
+
+        # Sanity check
+        if sum([len(group) for _, group in grouped_df]) < len(df_samples):
+            msg = "Some samples were not included in any submanifest."
```
Comment (ssjunnebo) [inline taca/element/Element_Runs.py:489]: Very nice check to have! 👍

**Review by ssjunnebo — APPROVED** [review]
Nice, thanks!

File: taca/element/Element_Runs.py (hunk 1 above)
Comment (kedhammar) [reply taca/element/Element_Runs.py:382]: If redemux is done manually, how about overriding the demux file path as a CLI parameter?

Comment (ssjunnebo) [reply taca/element/Element_Runs.py:382]: Yeah I think that would make sense 👍

**Pull request merged**